src/app/main/(tabs)/home/page.tsx

```diff
@@ -40,7 +43,28 @@ export default function DailyPage() {
     queryFn: getDailyTestInfo,
   });
 
-  if (isLoading || dailyTestQuery.isLoading) {
+  const dailyEnumLevels = useQuery({
```

Comment:
When naming enums, it’s generally more readable to start with the broader, descriptive context and end with the more specific purpose or type. So, DailyLevelsEnum is likely clearer and more intuitive because it reads like “an enum of daily levels,” making its purpose more obvious at a glance.